core/src/scss/elements/input/_radio-checkbox.scss

```diff
@@ -64,7 +64,6 @@
 
 [type='checkbox']:checked + label::before,
 [type='checkbox']:checked:disabled + label::before {
-  background: url("#{$su-image-path}/check.png");
```

Comment:
I'm pretty sure this is a fallback technique for browsers that don't support SVG backgrounds. Let me check on this.

Reply:
it looks widely supported https://caniuse.com/#feat=svg-css